mxcube3/routes/Diffractometer.py

```diff
@@ -93,7 +93,9 @@ def md_in_plate_mode():
 def get_movables_state():
     ret = {}
     for movable in mxcube.diffractometer.centring_motors_list:
-        ret.update(Utils.get_movable_state_and_position(movable))
+        motor_info = Utils.get_movable_state_and_position(movable)
+	if motor_info[movable]['position']:
```

Comment:
I do not understand why we need this test. Moreover, if `motor_info[movable]['position']` is 0, the `ret` dict won't be updated, which is certainly not what we want ?